<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WareWidget</class>
 <widget class="QWidget" name="WareWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>416</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="WareFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="openfluid::ui::common::ClickableLabel" name="CustomColor">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="IDButtonsLayout" stretch="1,0,0,0,0">
          <item>
           <widget class="openfluid::ui::common::ElidedLabel" name="IDLabel">
            <property name="styleSheet">
             <string notr="true">font:bold;</string>
            </property>
            <property name="text">
             <string notr="true">ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GenerateSrcButton">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="DocButton">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Open documentation</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="OrderLabel">
              <property name="font">
               <font>
                <weight>50</weight>
                <italic>true</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string notr="true">#</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="SpeedIconLabel">
                 <property name="minimumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>SpeedIconLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="DebugIconLabel">
                 <property name="minimumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>DebugIconLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="EnabledCheckBox">
              <property name="text">
               <string>Enabled</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="ColorButton">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="UpButton">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Move up</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="DownButton">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Move down</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="RemoveButton">
              <property name="minimumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Remove</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
          <item>
           <widget class="openfluid::ui::common::ElidedLabel" name="NameLabel">
            <property name="text">
             <string notr="true">NameLabel</string>
            </property>
            <property name="indent">
             <number>12</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="openfluid::ui::common::ActionLabel" name="ShowHideParamsLabel">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string notr="true">ShowHideParamsLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="ParamInfoWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Parameters:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="openfluid::ui::common::ActionLabel" name="ParameterizationSwitchLabel">
               <property name="text">
                <string notr="true">ParameterizationSwitchLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSplitter" name="ParamInfoSplitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QWidget" name="ParamsSideWidget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QStackedWidget" name="ParameterizationStackWidget">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="ParamsListPage">
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QScrollArea" name="ParamsListArea">
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>250</height>
                      </size>
                     </property>
                     <property name="baseSize">
                      <size>
                       <width>200</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::NoFrame</enum>
                     </property>
                     <property name="horizontalScrollBarPolicy">
                      <enum>Qt::ScrollBarAlwaysOff</enum>
                     </property>
                     <property name="widgetResizable">
                      <bool>true</bool>
                     </property>
                     <widget class="QWidget" name="ParamsListAreaContents">
                      <property name="geometry">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>527</width>
                        <height>250</height>
                       </rect>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>250</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="baseSize">
                       <size>
                        <width>250</width>
                        <height>0</height>
                       </size>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,1">
                       <item>
                        <widget class="QWidget" name="ParamInfoTitleWidget" native="true">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>0</height>
                          </size>
                         </property>
                         <layout class="QHBoxLayout" name="horizontalLayout_4">
                          <property name="leftMargin">
                           <number>0</number>
                          </property>
                          <property name="topMargin">
                           <number>0</number>
                          </property>
                          <property name="rightMargin">
                           <number>0</number>
                          </property>
                          <property name="bottomMargin">
                           <number>0</number>
                          </property>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <widget class="QWidget" name="ParamsListZoneWidget" native="true">
                         <layout class="QVBoxLayout" name="verticalLayout_6">
                          <property name="leftMargin">
                           <number>0</number>
                          </property>
                          <property name="rightMargin">
                           <number>0</number>
                          </property>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                         <item>
                          <spacer name="horizontalSpacer">
                           <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                           </property>
                           <property name="sizeHint" stdset="0">
                            <size>
                             <width>40</width>
                             <height>20</height>
                            </size>
                           </property>
                          </spacer>
                         </item>
                         <item>
                          <widget class="QPushButton" name="AddParamButton">
                           <property name="minimumSize">
                            <size>
                             <width>24</width>
                             <height>24</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>24</width>
                             <height>24</height>
                            </size>
                           </property>
                           <property name="toolTip">
                            <string>Add parameter</string>
                           </property>
                           <property name="text">
                            <string notr="true"/>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <spacer name="verticalSpacer">
                         <property name="orientation">
                          <enum>Qt::Vertical</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>20</width>
                           <height>40</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="SignatureWidget" name="InfosSideWidget" native="true"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignatureWidget</class>
   <extends>QWidget</extends>
   <header>SignatureWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>openfluid::ui::common::ElidedLabel</class>
   <extends>QLabel</extends>
   <header location="global">openfluid/ui/common/ElidedLabel.hpp</header>
  </customwidget>
  <customwidget>
   <class>openfluid::ui::common::ActionLabel</class>
   <extends>QLabel</extends>
   <header location="global">openfluid/ui/common/ActionLabel.hpp</header>
  </customwidget>
  <customwidget>
   <class>openfluid::ui::common::ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">openfluid/ui/common/ClickableLabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
